.github/workflows/pulsar-ci.yaml

```diff
@@ -620,10 +620,14 @@ jobs:
         run: |
           # build docker image
           # include building of Pulsar SQL, Connectors, Offloaders and server distros
-          mvn -B -am -pl pulsar-sql/presto-distribution,distribution/io,distribution/offloaders,distribution/server,tests/docker-images/latest-version-image install \
+          mvn -B -am -pl pulsar-sql/presto-distribution,distribution/io,distribution/offloaders,distribution/server,distribution/shell,tests/docker-images/latest-version-image install \
           -Pmain,docker -Dmaven.test.skip=true -Ddockerfile.build.squash=true \
           -Dspotbugs.skip=true -Dlicense.skip=true -Dcheckstyle.skip=true -Drat.skip=true
 
+      # check full build artifacts licenses
+      - name: Check binary licenses
+        run: src/check-binary-license.sh ./distribution/server/target/apache-pulsar-*-bin.tar.gz && src/check-binary-license.sh ./distribution/shell/target/apache-pulsar-shell-*-bin.tar.gz
```

Comment:
`apache-pulsar-*-bin.tar.gz ` why * can't match `apache-pulsar-shell-*-bin.tar.gz`

Reply:
Oh, I think it can match, but only in a different path.

Reply:
Ah, sorry I ignored it. can we give the path a `*`?

Reply:
Put a `*` on the path will match offloader and io, offloader and io are `nar`, license check with `nar` has to do more different things, we can separate this part.